### PR TITLE
Adding some channels: Seem to play well

### LINKED ISSUE #3763: Add: NBC 12 Phoenix (KPNX)
**Channel Name:** NBC 12 Phoenix (KPNX)
**Country:** US
**Language:** English
**Website:** https://www.12news.com/
**Stream URL (optional):** https://www.12news.com/watch
**Notes (optional):** Stream plays at 1080p in VLC and FFplay, Firefox browser tab.
https://livevideo01.12news.com/hls/live/2015501/newscasts/live.m3u8

### LINKED ISSUE #3764: Add: FOX 54.1 Huntsville AL (WZDX)
**Channel Name:** FOX 54.1 Huntsville AL (WZDX)
**Country:** US
**Language:** English
**Website:** https://www.rocketcitynow.com
**Stream URL (optional):** https://livevideo01.rocketcitynow.com/hls/live/2011659/newscasts/live.m3u8
**Notes (optional):** Streams at 1080p, VLC

### LINKED ISSUE #3765: Add: CBS 5 Ft. Smith, AR (KFSM)
**Channel Name:** CBS 5 Ft. Smith, AR (KFSM)
**Country:** US
**Language:** English
**Website:** https://www.5newsonline.com
**Stream URL (optional):** https://www.5newsonline.com/watch
**Notes (optional):** https://livevideo01.5newsonline.com/hls/live/2011653/newscasts/live.m3u8

### LINKED ISSUE #3766: Add: CBS THV11 Little Rock, AR (KTHV)
**Channel Name:** CBS THV11 Little Rock, AR (KTHV)
**Country:** US
**Language:** English
**Website:** https://www.thv11.com
**Stream URL (optional):** https://www.thv11.com/watch
**Notes (optional):** https://livevideo01.thv11.com/hls/live/2017154/newscasts/live.m3u8

### LINKED ISSUE #3767: Add: ABC 10 Sacramento, CA (KXTV)
**Channel Name:** ABC 10 Sacramento, CA (KXTV)
**Country:** US
**Language:** English
**Website:** https://www.abc10.com
**Stream URL (optional):** https://www.abc10.com/watch
**Notes (optional):** https://livevideo01.abc10.com/hls/live/2014547/newscasts/live.m3u8

### LINKED ISSUE #3768: Add: CW Network San Diego
**Channel Name:** CW Network San Diego 
**Country:** US
**Language:** English
**Website:** https://www.cbs8.com/cwnetwork/
**Stream URL (optional):** https://livevideo01.cbs8.com/hls/live/2014967/newscasts/live.m3u8
**Notes (optional):** Works in VLC, FFplay

### LINKED ISSUE #3769: Add: NBC 9 Denver, CO (KUSA)
**Channel Name:** NBC 9 Denver, CO (KUSA)
**Country:** US
**Language:** English
**Website:** https://www.9news.com
**Stream URL (optional):** https://www.9news.com/watch
**Notes (optional):** https://livevideo01.9news.com/hls/live/2014548/newscasts/live.m3u8

### LINKED ISSUE #3770: Add: FOX 61 Hartford, CT (WTIC)
**Channel Name:** FOX 61 Hartford, CT (WTIC)
**Country:** US
**Language:** English
**Website:** https://www.fox61.com
**Stream URL (optional):** https://www.fox61.com/watch
**Notes (optional):** https://livevideo01.fox61.com/hls/live/2011658/newscasts/live.m3u8

### LINKED ISSUE #3771: Add: NBC / ABC  Jacksonville, FL WTLV / WJXX
**Channel Name:** NBC / ABC  Jacksonville, FL WTLV / WJXX
**Country:** US
**Language:** English
**Website:** https://www.firstcoastnews.com
**Stream URL (optional):** https://www.firstcoastnews.com/watch
**Notes (optional):** https://livevideo01.firstcoastnews.com/hls/live/2014550/newscasts/live.m3u8

### LINKED ISSUE #3772: Add: CBS 10 Tampa, FL (WTSP)
**Channel Name:** CBS 10 Tampa, FL (WTSP)
**Country:** US
**Language:** English
**Website:** https://www.wtsp.com
**Stream URL (optional):** https://www.wtsp.com/watch
**Notes (optional):** https://livevideo01.wtsp.com/hls/live/2015503/newscasts/live.m3u8

### LINKED ISSUE #3773: Add: NBC 11 Alive Atlanta, GA  (WXIA)
**Channel Name:** NBC 11 Alive Atlanta, GA  (WXIA)
**Country:** US
**Language:** English
**Website:** https://www.11alive.com
**Stream URL (optional):** https://www.11alive.com/watch
**Notes (optional):** https://livevideo01.11alive.com/hls/live/2015499/newscasts/live.m3u8

### LINKED ISSUE #3774: Add: CBS 13 Macon, GA (WMAZ)
**Channel Name:** CBS 13 Macon, GA (WMAZ)
**Country:** US
**Language:** English
**Website:** https://www.13wmaz.com
**Stream URL (optional):** https://www.13wmaz.com/watch
**Notes (optional):** https://livevideo01.13wmaz.com/hls/live/2017376/newscasts/live.m3u8

### LINKED ISSUE #3775: Add: ABC 8 Davenport, IA (WQAD)
**Channel Name:** ABC 8 Davenport, IA (WQAD)
**Country:** US
**Language:** English
**Website:** https://www.wqad.com
**Stream URL (optional):** https://www.wqad.com/watch
**Notes (optional):** https://livevideo01.wqad.com/hls/live/2011657/newscasts/live.m3u8

### LINKED ISSUE #3776: Add: ABC 5 Des Moines, IA (WOI)
**Channel Name:** ABC 5 Des Moines, IA (WOI)
**Country:** US
**Language:** English
**Website:** https://www.weareiowa.com
**Stream URL (optional):** https://www.weareiowa.com/watch
**Notes (optional):** https://livevideo01.weareiowa.com/hls/live/2011593/newscasts/live.m3u8

### LINKED ISSUE #3777: Add: KTVB 7 Boise, ID
**Channel Name:** KTVB 7 Boise, ID
**Country:** US
**Language:** English
**Website:** https://www.ktvb.com
**Stream URL (optional):** https://www.ktvb.com/watch
**Notes (optional):** https://livevideo01.ktvb.com/hls/live/2014542/newscasts/live.m3u8

### LINKED ISSUE #3778: Add: NBC 13 Indianapolis, IN (WTHR)
**Channel Name:** NBC 13 Indianapolis, IN (WTHR)
**Country:** US
**Language:** English
**Website:** https://www.wthr.com
**Stream URL (optional):** https://www.wthr.com/watch
**Notes (optional):** https://livevideo01.wthr.com/hls/live/2013835/newscasts/live.m3u8

### LINKED ISSUE #3779: Add: ABC 11 Louisville, KY (WHAS)
**Channel Name:** ABC 11 Louisville, KY (WHAS)
**Country:** US
**Language:** English
**Website:** https://www.whas11.com
**Stream URL (optional):** https://www.whas11.com/watch
**Notes (optional):** https://livevideo01.whas11.com/hls/live/2016284/newscasts/live.m3u8

### LINKED ISSUE #3780: Add: CBS 4 New Orleans, LA (WWL)
**Channel Name:** CBS 4 New Orleans, LA (WWL)
**Country:** US
**Language:** English
**Website:** https://www.wwltv.com
**Stream URL (optional):** https://www.wwltv.com/watch
**Notes (optional):** https://livevideo01.wwltv.com/hls/live/2016516/newscasts/live.m3u8

### LINKED ISSUE #3781: Add: NBC 207  Bangor / Portland, ME (WLBZ / WCSH)
**Channel Name:** NBC 207  Bangor / Portland, ME (WLBZ / WCSH)
**Country:** US
**Language:** English
**Website:** https://www.newscentermaine.com
**Stream URL (optional):** https://www.newscentermaine.com/watch
**Notes (optional):** https://livevideo01.newscentermaine.com/hls/live/2014540/newscasts/live/wcsh.m3u8

### LINKED ISSUE #3782: Add: ABC 13 Grand Rapids, MI (WZZM)
**Channel Name:** ABC 13 Grand Rapids, MI (WZZM)
**Country:** US
**Language:** English
**Website:** https://www.wzzm13.com
**Stream URL (optional):** https://www.wzzm13.com/watch
**Notes (optional):** https://livevideo01.wzzm13.com/hls/live/2016280/newscasts/live.m3u8

### LINKED ISSUE #3783: Add: NBC 11 Minneapolis, MN (KARE)
**Channel Name:** NBC 11 Minneapolis, MN (KARE)
**Country:** US
**Language:** English
**Website:** https://www.kare11.com
**Stream URL (optional):** https://www.kare11.com/watch
**Notes (optional):** https://livevideo01.kare11.com/hls/live/2014544/newscasts/live.m3u8

### LINKED ISSUE #3784: Add: NBC 36 Charlotte, NC (WCNC)
**Channel Name:** NBC 36 Charlotte, NC (WCNC)
**Country:** US
**Language:** English
**Website:** https://www.wcnc.com
**Stream URL (optional):** https://www.wcnc.com/watch
**Notes (optional):** https://livevideo01.wcnc.com/hls/live/2015505/newscasts/live.m3u8

### LINKED ISSUE #3785: Add: CBS 2 Greensboro, NC (WFMY)
**Channel Name:** CBS 2 Greensboro, NC (WFMY)
**Country:** US
**Language:** English
**Website:** https://www.wfmynews2.com
**Stream URL (optional):** https://www.wfmynews2.com/watch
**Notes (optional):** https://livevideo01.wfmynews2.com/hls/live/2016285/newscasts/live.m3u8

### LINKED ISSUE #3786: Add: NBC 2 Buffalo, NY (WGRZ)
**Channel Name:** NBC 2 Buffalo, NY (WGRZ)
**Country:** US
**Language:** English
**Website:** https://www.wgrz.com
**Stream URL (optional):** https://www.wgrz.com/watch
**Notes (optional):** https://livevideo01.wgrz.com/hls/live/2016286/newscasts/live.m3u8

### LINKED ISSUE #3787: Add: 3 Studios Cleveland, OH (WKYC)
**Channel Name:** 3 Studios Cleveland, OH (WKYC)
**Country:** US
**Language:** English
**Website:** https://www.wkyc.com
**Stream URL (optional):** https://www.wkyc.com/watch
**Notes (optional):** https://livevideo01.wkyc.com/hls/live/2015504/newscasts/live.m3u8

### LINKED ISSUE #3788: Add: CBS 10 Columbus, OH (WBNS)
**Channel Name:** CBS 10 Columbus, OH (WBNS)
**Country:** US
**Language:** English
**Website:** https://www.10tv.com
**Stream URL (optional):** https://www.10tv.com/watch
**Notes (optional):** https://livevideo01.10tv.com/hls/live/2013836/newscasts/live.m3u8

### LINKED ISSUE #3789: Add: CBS 11 Toledo, OH (WTOL)
**Channel Name:** CBS 11 Toledo, OH (WTOL)
**Country:** US
**Language:** English
**Website:** https://www.wtol.com
**Stream URL (optional):** https://www.wtol.com/watch
**Notes (optional):** https://livevideo01.wtol.com/hls/live/2017153/newscasts/live.m3u8

### LINKED ISSUE #3790: Add: NBC 8 Portland, OR (KGW)
**Channel Name:** NBC 8 Portland, OR (KGW)
**Country:** US
**Language:** English
**Website:** https://www.kgw.com
**Stream URL (optional):** https://www.kgw.com/watch
**Notes (optional):** https://livevideo01.kgw.com/hls/live/2015506/newscasts/live.m3u8

### LINKED ISSUE #3792: Add: FOX 43 Harrisburg, PA (WPMT)
**Channel Name:** FOX 43 Harrisburg, PA (WPMT)
**Country:** US
**Language:** English
**Website:** https://www.fox43.com
**Stream URL (optional):** https://www.fox43.com/watch
**Notes (optional):** https://livevideo01.fox43.com/hls/live/2011656/newscasts/live.m3u8

### LINKED ISSUE #3793: Add: ABC 16 Wilkes Barre, PA (WNEP)
**Channel Name:** ABC 16 Wilkes Barre, PA (WNEP)
**Country:** US
**Language:** English
**Website:** https://www.wnep.com
**Stream URL (optional):** https://www.wnep.com/watch
**Notes (optional):** https://livevideo01.wnep.com/hls/live/2011655/newscasts/live.m3u8

### LINKED ISSUE #3794: Add: CBS 19 Columbia, SC (WLTX)
**Channel Name:** CBS 19 Columbia, SC (WLTX)
**Country:** US
**Language:** English
**Website:** https://www.wltx.com
**Stream URL (optional):** https://www.wltx.com/watch
**Notes (optional):** https://livevideo01.wltx.com/hls/live/2017152/newscasts/live.m3u8

### LINKED ISSUE #3795: Add: NBC 10 Knoxville, TN (WBIR)
**Channel Name:** NBC 10 Knoxville, TN (WBIR)
**Country:** US
**Language:** English
**Website:** https://www.wbir.com
**Stream URL (optional):** https://www.wbir.com/watch
**Notes (optional):** https://livevideo01.wbir.com/hls/live/2016515/newscasts/live-2000.m3u8

### LINKED ISSUE #3796: Add: ABC 24 Memphis, TN (WATN)
**Channel Name:** ABC 24 Memphis, TN (WATN)
**Country:** US
**Language:** English
**Website:** https://www.localmemphis.com
**Stream URL (optional):** https://www.localmemphis.com/watch
**Notes (optional):** https://livevideo01.localmemphis.com/hls/live/2011654/newscasts/live.m3u8

### LINKED ISSUE #3797: Add: FOX 15 West Texas Abilene, TX (KXVA-TV)
**Channel Name:** FOX 15 West Texas Abilene, TX (KXVA-TV)
**Country:** US
**Language:** English
**Website:** https://www.myfoxzone.com
**Stream URL (optional):** https://www.myfoxzone.com/watch
**Notes (optional):** https://livevideo01.myfoxzone.com/hls/live/2017381/newscasts/live.m3u8

### LINKED ISSUE #3798: Add: ABC 24 Austin, TX (KVUE)
**Channel Name:** ABC 24 Austin, TX (KVUE)
**Country:** US
**Language:** English
**Website:** https://www.kvue.com
**Stream URL (optional):** https://www.kvue.com/watch
**Notes (optional):** https://livevideo01.kvue.com/hls/live/2016282/newscasts/live.m3u8

### LINKED ISSUE #3799: Add: 12 News Beaumont, TX (KBMT)
**Channel Name:** 12 News Beaumont, TX (KBMT)
**Country:** US
**Language:** English
**Website:** https://www.12newsnow.com
**Stream URL (optional):** https://www.12newsnow.com/watch
**Notes (optional):** https://livevideo01.12newsnow.com/hls/live/2017379/newscasts/live.m3u8

### LINKED ISSUE #3800: Add: NBC 6 College Station, TX (KAGS)
**Channel Name:** NBC 6 College Station, TX (KAGS)
**Country:** US
**Language:** English
**Website:** https://www.kagstv.com
**Stream URL (optional):** https://www.kagstv.com/watch
**Notes (optional):** https://livevideo01.kagstv.com/hls/live/2016283/newscasts/live.m3u8

### LINKED ISSUE #3801: Add: ABC 3 Corpus Christi, TX (KIII)
**Channel Name:** ABC 3 Corpus Christi, TX (KIII)
**Country:** US
**Language:** English
**Website:** https://www.kiiitv.com
**Stream URL (optional):** https://www.kiiitv.com/watch
**Notes (optional):** https://livevideo01.kiiitv.com/hls/live/2017378/newscasts/live.m3u8

### LINKED ISSUE #3802: Add: ABC 8 Dallas, TX (WFAA)
**Channel Name:** ABC 8 Dallas, TX (WFAA)
**Country:** US
**Language:** English
**Website:** https://www.wfaa.com
**Stream URL (optional):** https://www.wfaa.com/watch
**Notes (optional):** https://livevideo01.wfaa.com/hls/live/2014541/newscasts/live.m3u8

### LINKED ISSUE #3803: Add: CBS 11 Houston, TX (KHOU)
**Channel Name:** CBS 11 Houston, TX (KHOU)
**Country:** US
**Language:** English
**Website:** https://www.khou.com
**Stream URL (optional):** https://www.khou.com/watch
**Notes (optional):** https://livevideo01.khou.com/hls/live/2014966/newscasts/live.m3u8

### LINKED ISSUE #3804: Add: NBC NewsWest 9 Midland-Odessa, TX (KWES)
**Channel Name:** NBC NewsWest 9 Midland-Odessa, TX (KWES)
**Country:** US
**Language:** English
**Website:** https://www.newswest9.com
**Stream URL (optional):** https://www.newswest9.com/watch
**Notes (optional):** https://livevideo01.newswest9.com/hls/live/2017380/newscasts/live.m3u8

### LINKED ISSUE #3805: Add: CBS 5 San Antonio, TX (KENS)
**Channel Name:** CBS 5 San Antonio, TX (KENS)
**Country:** US
**Language:** English
**Website:** https://www.kens5.com
**Stream URL (optional):** https://www.kens5.com/watch
**Notes (optional):** https://livevideo01.kens5.com/hls/live/2016281/newscasts/live.m3u8

### LINKED ISSUE #3806: Add: CBS 19  Tyler, TX (KYTX)
**Channel Name:** CBS 19  Tyler, TX (KYTX)
**Country:** US
**Language:** English
**Website:** https://www.cbs19.tv
**Stream URL (optional):** https://www.cbs19.tv/watch
**Notes (optional):** https://livevideo01.cbs19.tv/hls/live/2017377/newscasts/live.m3u8

### LINKED ISSUE #3807: Add: NBC 6 Waco, TX (KCEN)
**Channel Name:** NBC 6 Waco, TX (KCEN)
**Country:** US
**Language:** English
**Website:** https://www.kcentv.com
**Stream URL (optional):** https://www.kcentv.com/watch
**Notes (optional):** https://livevideo01.kcentv.com/hls/live/2017155/newscasts/live.m3u8

### LINKED ISSUE #3808: Add: ABC 13 Norfolk, VA (WVEC)
**Channel Name:** ABC 13 Norfolk, VA (WVEC)
**Country:** US
**Language:** Engilsh
**Website:** https://www.13newsnow.com
**Stream URL (optional):** https://www.13newsnow.com/watch
**Notes (optional):** https://livevideo01.13newsnow.com/hls/live/2014545/newscasts/live.m3u8

### LINKED ISSUE #3809: Add: NBC 5 Seattle, WA (KING)
**Channel Name:** NBC 5 Seattle, WA (KING)
**Country:** US
**Language:** English
**Website:** https://www.king5.com
**Stream URL (optional):** https://www.king5.com/watch
**Notes (optional):** https://livevideo01.king5.com/hls/live/2006665/live/live.m3u8

### LINKED ISSUE #3810: Add: CBS 2 Spokane, WA (KREM)
**Channel Name:** CBS 2 Spokane, WA (KREM)
**Country:** US
**Language:** English
**Website:** https://www.krem.com
**Stream URL (optional):** https://www.krem.com/watch
**Notes (optional):** https://livevideo01.krem.com/hls/live/2017156/newscasts/live.m3u8

### PR DESCRIPTION
**Pull request by jkemp814**
Please test, I can add to this PR, if they prove to play well for you all. 

Plus, I have about 26 more I hadn't listed on the issues page.

<details>
  <summary>Closes</summary>
Closes #3763
Closes #3764 
Closes #3765 
Closes #3766 
Closes #3767 
Closes #3768 
Closes #3769 
Closes #3770 
Closes #3771 
Closes #3772 
Closes #3773 
Closes #3774 
Closes #3775 
Closes #3776 
Closes #3777 
Closes #3778 
Closes #3779 
Closes #3780  
Closes #3781 
Closes #3782 
Closes #3783 
Closes #3784 
Closes #3785 
Closes #3786 
Closes #3787 
Closes #3788 
Closes #3789 
Closes #3790 
Closes #3792 
Closes #3793 
Closes #3794 
Closes #3795 
Closes #3796 
Closes #3797 
Closes #3798 
Closes #3799 
Closes #3800 
Closes #3801 
Closes #3802 
Closes #3803 
Closes #3804 
Closes #3805 
Closes #3806 
Closes #3807 
Closes #3808 
Closes #3809 
Closes #3810</details>

Will add logos if channels check out.

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
Wow, that's a nice bunch of channels. Let me see.

File: channels/us.m3u [hunk 1]
```diff
@@ -6526,3 +6526,59 @@ https://ln-zen.localnowlive.com/v1/master/385c85a93929f94966d0fb186fc33b431e6f1e
...
+https://livevideo01.11alive.com/hls/live/2015499/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="WTSP.us" tvg-name="CBS 10 Tampa FL (WTSP)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",CBS 10 Tampa, FL (WTSP)
+https://livevideo01.wtsp.com/hls/live/2015503/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="WTLV.us" tvg-name="NBC Jacksonville FL (WTLV)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",NBC / ABC Jacksonville, FL (WTLV)
+https://livevideo01.firstcoastnews.com/hls/live/2014550/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="WTIC.us" tvg-name="FOX 61 Hartford CT (WTIC)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",FOX 61 Hartford, CT (WTIC)
+https://livevideo01.fox61.com/hls/live/2011658/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="KUSA.us" tvg-name="NBC 9 Denver CO (KUSA)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",NBC 9 Denver, CO (KUSA)
+https://livevideo01.9news.com/hls/live/2014548/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="CWNetworkSanDiego.us" tvg-name="CW Network San Diego" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",CW Network San Diego
```
Comment (LaneSh4d0w) [inline channels/us.m3u:6573]: Is it really the CW Network ? I see CBS 8 San Diego (KFMB)

Comment (jkemp814) [reply channels/us.m3u:6573]: I just rename it. It really channel 8.2 KFMB-TV2. The Grit was like that yesterday.

File: channels/us.m3u [hunk 2]
```diff
@@ -6526,3 +6526,60 @@ https://ln-zen.localnowlive.com/v1/master/385c85a93929f94966d0fb186fc33b431e6f1e
...
+https://livevideo01.abc10.com/hls/live/2014547/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="KTHV.us" tvg-name="CBS THV11 Little Rock AR (KTHV)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",CBS THV11 Little Rock, AR (KTHV)
+https://livevideo01.thv11.com/hls/live/2017154/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="KFSM.us" tvg-name="CBS 5 Ft. Smith AR (KFSM)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",CBS 5 Ft. Smith, AR (KFSM)
+https://livevideo01.5newsonline.com/hls/live/2011653/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="WZDX.us" tvg-name="FOX 54.1 Huntsville AL (WZDX)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",FOX 54.1 Huntsville AL (WZDX)
+https://livevideo01.rocketcitynow.com/hls/live/2011659/newscasts/live.m3u8
+#EXTINF:-1 tvg-id="KPNX.us" tvg-name="NBC 12 Phoenix (KPNX)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",NBC 12 Phoenix (KPNX)
+https://livevideo01.12news.com/hls/live/2015501/newscasts/live.m3u8
+
```
Comment (LaneSh4d0w) [inline channels/us.m3u:6585]: Remove the additional line.

**Comment by LaneSh4d0w** [comment]
I agree, all channels work perfectly on my watch.

**Comment by jkemp814** [comment]
They all play at 1080p here.

**Comment by LaneSh4d0w** [comment]
Same on my side. PR is good to go. Once this PR is R2R, i'll merge.

**Comment by jkemp814** [comment]
imgur server going wacky. Not letting me upload right now, saying try again later.

**Review by MapGuy11 — APPROVED** [review]

**Comment by MapGuy11** [comment]
Can I merge? Are you all done?

**Comment by jkemp814** [comment]
Yes I'm done. I have a few more, but going to have to do more research. So let's merge this one, please.

**Comment by MapGuy11** [comment]
> Yes I'm done. I have a few more, but going to have to do more research. So let's merge this one, please.

Which ones are they maybe I can have a look.

**Comment by jkemp814** [comment]
They are sub-channels. The links I have keep defaulting to the top channel. I will make a list and post it.

**Comment by LaneSh4d0w** [comment]
> 
> 
> > Yes I'm done. I have a few more, but going to have to do more research. So let's merge this one, please.
> 
> Which ones are they maybe I can have a look.

I think @jkemp814 was searching to add the ABC 30 Saint Louis channel to the list. I've tried looking at it but it's on a tokened Uplynk stream.

**Comment by MapGuy11** [comment]
> > > Yes I'm done. I have a few more, but going to have to do more research. So let's merge this one, please.
> > 
> > 
> > Which ones are they maybe I can have a look.
> 
> I think @jkemp814 was searching to add the ABC 30 Saint Louis channel to the list. I've tried looking at it but it's on a tokened Uplynk stream.

From the looks of it they don't have a news operation so it would just be a tokened full stream. Not like these where they only show when they have newscasts.

**Pull request merged**